### PR TITLE
Remove erroneous attack traits from some spells

### PR DESCRIPTION
**Pull request by pedrogrullada**
and remove some seemingly accidental bolded text description here and there.

### CODE REVIEW

File: packs/spells/vector-screen.json [hunk 1]
```diff
@@ -25,12 +25,14 @@
...
                         "areaType": "square",
+                        "type": "burst",
                         "value": "10"
                     }
                 },
                 "6": {
                     "area": {
                         "areaType": "square",
+                        "type": "burst",
                         "value": "10"
```
Comment (TikaelSol) [inline packs/spells/vector-screen.json:36]: This looks like a bug in the heightening overlay. @CarlosFdez any thoughts?

Comment (stwlam) [reply packs/spells/vector-screen.json:36]: I think it's legacy data that slipped in. It's be good to remove it.

Comment (pedrogrullada) [reply packs/spells/vector-screen.json:36]: Anything I should do with this? It came out like that by itself so I assumed it was something the system did when I unlocked the compendium to edit the spell. It looked correct according to the spell's description, so I left it alone.

Comment (CarlosFdez) [reply packs/spells/vector-screen.json:36]: Value and type are the correct fields, so areaType is legacy data. I don't see any reference to it in the templates.

File: packs/spells/vector-screen.json [hunk 2]
```diff
@@ -25,12 +25,14 @@
                 "5": {
                     "area": {
                         "areaType": "square",
```
Comment (stwlam) [inline packs/spells/vector-screen.json:27]:
```suggestion
```

File: packs/spells/vector-screen.json [hunk 3]
```diff
@@ -25,12 +25,14 @@
...
+                        "type": "burst",
                         "value": "10"
                     }
                 },
                 "6": {
                     "area": {
                         "areaType": "square",
```
Comment (stwlam) [inline packs/spells/vector-screen.json:34]:
```suggestion
```

**Pull request merged**